featuretools/tests/synthesis/test_dask_dfs.py

```diff
@@ -55,7 +55,7 @@ def test_single_table_dask_entityset():
 
     # Use the same columns and make sure both indexes are sorted the same
     dask_computed_fm = dask_fm.compute().set_index('id').loc[fm.index][fm.columns]
-    pd.testing.assert_frame_equal(fm, dask_computed_fm)
+    pd.testing.assert_frame_equal(fm, dask_computed_fm, check_dtype=False)
```

Comment:
I updated these test checks to ignore the dtypes, because this update means we can no longer expect the dask and koalas feature matrices to have the same dtypes for all the columns. With pandas, the logical types would only change when they need to, but with Dask and Koalas they would always end up using the nullable types in the feature matrix.